guides/common/modules/proc_generating-host-monitoring-reports.adoc

```diff
@@ -31,3 +31,26 @@ If you have selected the *Send report via e-mail* checkbox, the host monitoring
 +
 This command waits until the report fully generates before completing.
 If you want to generate the report as a background task, you can use the `hammer report-template schedule` command.
++
+[NOTE]
+====
+If you want to generate a subscription entitlement report, you have to use the `Days from Now` option to specify the latest expiration time of entitlement subscriptions.
+You can use the `no limit` value to show all entitlements.
+See the examples below.
+
+.Show all entitlements
+[options="nowrap", subs="+quotes,attributes,verbatim"]
+----
+# hammer report-template generate \
+--name 'Subscription - Entitlement Report' \
+--inputs "Days from Now=no limit"
+----
+
+.Show all entitlements that are going to expire within 60 days
+[options="nowrap", subs="+quotes,attributes,verbatim"]
+----
+# hammer report-template generate \
+--name 'Subscription - Entitlement Report' \
+--inputs "Days from Now=60"
```

Comment:
```suggestion
--inputs "Days from Now=60" \
--name "Subscription - Entitlement Report"
```

Reply:
Starting with name here seems the most logical to me, why should this be switched?